docs/content/Coding Milestones/PoC2023q1/commands.md

```diff
@@ -608,24 +605,38 @@ KUBECONFIG=$imw_space_config kubectl kubestellar remove location demo1
 
 ## Getting a kubeconfig of a given space
 
-To access a particular space the kubeconfig file of that space needs to be
-obtained.  This can be done manually as follows: 
+This command fetches a kubeconfig for super-user access to a given
+space into a given filename. The kubeconfig will be good for one year
+since the creation of the space unless it was configured unusually.
+
+The usage synopsis is as follows.
 
 ```shell
-secret_name=$(kubectl --kubeconfig $sm_core_config --context $sm_context get space imw-1 -n spaceprovider-default -o jsonpath="{$.status.externalSecretRef.name}")
-secret_namespace=$(kubectl --kubeconfig $sm_core_config --context $sm_context get space imw-1 -n spaceprovider-default -o jsonpath="{$.status.externalSecretRef.namespace}")
-kubectl --kubeconfig $sm_core_config --context $sm_context get secret ${secret_name} -n ${secret_namespace} -o jsonpath='{$.data.kubeconfig}' | base64 -d | base64 -d > imw.kubeconfig
+kubectl kubestellar get config for space flag...
 ```
 
-Alternatively, the kubectl-kubestellar-get-config-for-space can be used. This script takes
-as input the name of the space (--space-name), the name of the space provider (--provider-name) 
-which has a default of "default", the location of the space manager 
-kubeconfig (--sm-core-config), the context of the space manager (--sm-context), whether the space 
-manager is accessed in-cluster or externally (in_cluster), and the name of the output kubeconfig 
-file (--output).
+This command accepts the following flags. The space name, output
+pathname, and SMAS kubeconfig and context are required.
+
+- `-h` or `--help`: print usage and terminate with success.
+- `-X`: turn on debug echoing of the script as it executes.
+- `--space-name $space_name`: the name of the space.
+- `--output $kubeconfig_pathname`: where to write the kubeconfig
+- `--provider-name $provider`: the name of the space provider to use;
+  defaults to `default`.
+- `--sm-core-config $SM_CONFIG`: pathname of the kubeconfig file for
+  access to the SMAS.
+- `--sm-context $SM_CONTEXT`: the name of the context to use in that
+  kubeconfig for access to the SMAS.
+- `--in-cluster`: when the core components are running in a hosting
```

Comment:
I think a simpler explanation might be "when the caller resides in the same hosting cluster as the space-provider"

Reply:
Remember that contributors can deploy the core stuff as bare processes, so it is possible that there is no hosting cluster.